stack-8.6.5.yaml

```diff
@@ -15,6 +15,7 @@ packages:
   - ./plugins/hls-explicit-imports-plugin
   - ./plugins/hls-refine-imports-plugin
   - ./plugins/hls-hlint-plugin
+  - ./plugins/hls-rename-plugin
```

Comment:
Since we don't expect the plugin to build with 8.6.x, should it be removed from this stack.yaml file?